tests/meson.build

```diff
@@ -11,7 +11,10 @@ test_src = files(
 test('all',
     executable(meson.project_name() + '_test',
         test_src,
-        dependencies : libgit4cpp_dep,
+        dependencies : [
+            dependency('catch2', version : '>3.4.0'),
```

Comment:
This is now not-in-sync with the `debian/control` anymore which requires `>= 3.4.0`

Reply:
Changed to `>=`. Thanks a lot, @Finii!